packages/manager/modules/freefax/src/freefax/credit/index.js

```diff
@@ -11,7 +11,7 @@ import template from './freefax-credit.html';
 const moduleName = 'managerFreefaxCredit';
 
 angular
-  .module(moduleName, ['ngOvhContracts', 'oui'])
+  .module(moduleName, ['ngOvhContracts', 'oui', 'ovh-api-services'])
```

Comment:
If we remove a package, why do we add the injection of `ovh-api-services`?

Reply:
As described in the PR body : I removed `@ovh-ux/manager-notifications-sidebar` usage, but it breaks the existing applications.
I think `ovh-api-services` was imported and inejected in `@ovh-ux/manager-notifications-sidebar` in early steps. 
As I removed it, the package is no more imported, so I fix the application.